seedsource_core/django/seedsource/management/commands/create_vector_tiles.py

```diff
@@ -0,0 +1,89 @@
+from django.core.management import BaseCommand
+from django.conf import settings
+from seedsource_core.django.seedsource.models import SeedZone
+from tempfile import mkdtemp
+import os
+import subprocess
+import json
+import shutil
+
+
+class Command(BaseCommand):
+    help = 'Facilitates converting of vector data into vector tiles.'
+
+    def _write_out(self, output):
+        self.stdout.write('\033[0;33m' + output + '\033[0m')
+
+    def _create_folder(self, directory):
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+
+    def handle(self, *args, **options):
+        tiles_dir = os.path.join(settings.BASE_DIR, "tiles")
+        outputIndex = []
+        errors = []
+        sources = SeedZone.objects.values_list('source').distinct()
+
+        self._create_folder(os.path.join(tiles_dir, "seedzones"))
+
+        for source in sources:
+            source = str(source).strip("'(,)'")
```

Comment:
`source` should already be a string.

Reply:
unfortunately it's a tuple... I replaced the garbage above with source[0]

Reply:
Ah, ok. Actually just add `flat=True` to the `values_list` call.

```python
sources = SeedZone.objects.values_list('source', flat=True).distinct()
```